src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, Alert, TouchableOpacity, FlatList } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, FlatButton, ButtonType, Image } from 'app/components';
+import { Route, MainCardStackNavigatorParams, RootStackParams, Wallet } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.Notifications>
+  >;
+  wallets: any;
```

Comment:
The same here - I'd change `any` to `unknown[]` at least but having a specifc type would be great

Reply:
done